ui_framework/components/local_nav/_index.scss

```diff
@@ -17,6 +17,12 @@ $localTabTextColor-isSelected: $localNavButtonTextColor-isHover;
 $localBreadcrumbLinkColor: #328caa;
 $localSearchButtonBackgroundColor: #9c9c9c;
 $localSearchButtonTextColor: #ffffff;
+$localCalendarButtonHoverBackgroundColor: $localNavButtonBackgroundColor-isHover;
+$localCalendarCellPrecludedBackgroundColor: rgba(black, 0.05);
+$localCalendarCellSelectedBackgroundColor: #777777;
+$localCalendarCellSelectedTextColor: #ffffff;
+$localCalendarCellHoverBackgroundColor: $globalInfoColor;
```

Comment:
Oh actually I was wrong. You _can_ select it again, which will reset the time. I'll make the change you suggested, thanks.